MibTeX/src/de/mibtex/BibtexEntry.java

```diff
@@ -278,4 +312,7 @@ public String toString() {
 				+ ", citations=" + citations + "]";
 	}
 
+	public static boolean IsDefined(String attribute) {
```

Comment:
Small first letter for method names

Reply:
Ok, will do. In general, I use capital letters for static methods.